projects/go-lib/src/lib/components/go-side-nav/go-side-nav/go-side-nav.component.scss

```diff
@@ -58,6 +58,7 @@
 }
 
 .go-side-nav__app-drawer {
+  border-radius: 50%;
```

Comment:
This will not work if the element is not a perfect square, which sometimes it is not in this case.

Reply:
![Screenshot (249)](https://user-images.githubusercontent.com/3825309/94144489-d1fc3780-fe8e-11ea-8720-7d05c46dde60.png)
If the hover state has to be a circle, isnt this the only way to get one?. 
I know it looks more like an ellipse if you look closely. 
Shall we use a different icon? because introducing a height and width here will have its own problems.

Reply:
**Update**: changing the `go-icon` to `go-icon-button`, inside the action sheet, did make the hover background as a perfect circle.

Reply:
Thank you @grahamhency , fixed this.